src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
+  const symbols = ['AC', '+/-', '%', '/', '7', '8', '9', 'X', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Calculator.js%20file.-,In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A,-Implement%20the%20design), Please replace the divide symbol you added `/` with this symbol `÷` instead to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/218836370-5286a6b0-3103-4941-aea2-12914937bb0e.png)  | ![image](https://user-images.githubusercontent.com/68796232/218836333-7e712f03-55f6-4363-a06d-c1aaeeece4dd.png)  |

- [ ] [Optional] There is no reason to declare the symbols array here and pass it as a prop to the `Calculator` component, It would be better professionally if you declare it inside the Calculator component itself.